File: vb/demo/IceStorm/clock/ClockI.vb
' **********************************************************************
'
' Copyright (c) 2003-2006 ZeroC, Inc. All rights reserved.
'
' This copy of Ice is licensed to you under the terms described in the
' ICE_LICENSE file included in this distribution.
'
' **********************************************************************

Imports Demo

Public Class ClockI
    Inherits ClockDisp_

    Public Overloads Overrides Sub tick(ByVal current As Ice.Current)
	System.Console.Out.WriteLine("tick")
    End Sub
End Class

File: cs/demo/IceStorm/clock/ClockI.cs
// **********************************************************************
//
// Copyright (c) 2003-2006 ZeroC, Inc. All rights reserved.
//
// This copy of Ice is licensed to you under the terms described in the
// ICE_LICENSE file included in this distribution.
//
// **********************************************************************

using Demo;

public class ClockI : ClockDisp_
{
    public override void tick(Ice.Current current)
    {
        System.Console.Out.WriteLine("tick");
    }
}
